modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
```

Comment:
@syedriko Please review and approve this next-to final draft.
@abrennan89 Please peer-review this topic.

Reply:
@anpingli  Please assign a QE to review this topic.

Reply:
@kabirbhartiRH  Please review this topic.

Reply:
@rolfedh Could you replace `aws` with `aws --output json`? That'll take care of the issue @kabirbhartiRH ran into.

Reply:
Updated. Thanks.